scripts/PokemonTower5F.asm

```diff
@@ -34,117 +35,118 @@ PokemonTower5Script0:
 	call Delay3
 	call Delay3
 	call GBFadeInFromWhite
-	ld a, $7
+	ld a, TEXT_POKEMONTOWER5F_PURIFIEDZONE
 	ldh [hSpriteIndexOrTextID], a
 	call DisplayTextID
 	xor a
 	ld [wJoyIgnore], a
 	ret
 
-CoordsData_60992:
+PokemonTower5FPurifiedZoneCoords:
 	dbmapcoord 10,  8
 	dbmapcoord 11,  8
 	dbmapcoord 10,  9
 	dbmapcoord 11,  9
 	db -1 ; end
 
 PokemonTower5F_TextPointers:
-	dw PokemonTower5Text1
-	dw PokemonTower5Text2
-	dw PokemonTower5Text3
-	dw PokemonTower5Text4
-	dw PokemonTower5Text5
-	dw PickUpItemText
-	dw PokemonTower5Text7
+	def_text_pointers
+	dw_const PokemonTower5FChanneler1Text,   TEXT_POKEMONTOWER5F_CHANNELER1
+	dw_const PokemonTower5FChanneler2Text,   TEXT_POKEMONTOWER5F_CHANNELER2
+	dw_const PokemonTower5FChanneler3Text,   TEXT_POKEMONTOWER5F_CHANNELER3
+	dw_const PokemonTower5FChanneler4Text,   TEXT_POKEMONTOWER5F_CHANNELER4
+	dw_const PokemonTower5FChanneler5Text,   TEXT_POKEMONTOWER5F_CHANNELER5
+	dw_const PickUpItemText,                 TEXT_POKEMONTOWER5F_NUGGET
```

Comment:
```asm
	dw_const PokemonTower5FChanneler1Text,  TEXT_POKEMONTOWER5F_CHANNELER1
	dw_const PokemonTower5FChanneler2Text,  TEXT_POKEMONTOWER5F_CHANNELER2
	dw_const PokemonTower5FChanneler3Text,  TEXT_POKEMONTOWER5F_CHANNELER3
	dw_const PokemonTower5FChanneler4Text,  TEXT_POKEMONTOWER5F_CHANNELER4
	dw_const PokemonTower5FChanneler5Text,  TEXT_POKEMONTOWER5F_CHANNELER5
	dw_const PickUpItemText,                TEXT_POKEMONTOWER5F_NUGGET
```

Reply:
I'm confused, what did you change here? Alignment? Cause these are currently aligned to the line below this that you didn't copy.

Reply:
Every line has one more space than it needs.

Reply:
Those lines are aligned to `dw_const PokemonTower5FPurifiedZoneText, TEXT_POKEMONTOWER5F_PURIFIEDZONE` below it. The purifiedzone line only has a single space between the comma and the TEXT_ constant. This is what i'm seeing:
![image](https://user-images.githubusercontent.com/6394873/215304884-f4be5d5b-6a3f-4063-9cf5-cc7dd346b446.png)

Reply:
Unless you *don't* want to align them to the last line... but I would disagree with that.